polling_stations/apps/file_uploads/views.py

```diff
@@ -403,70 +398,23 @@ def test_func(self):
         return self.request.user.is_staff
 
     def form_valid(self, form):
-        self.council: Council = Council.objects.get(
-            council_id=self.kwargs["council_id"]
-        )
-        csv_file = self.request.FILES["csv_file"]
-        self.handle_file(csv_file)
+        self.council: Council = Council.objects.prefetch_related(
+            "pollingstation_set"
+        ).get(council_id=self.kwargs["council_id"])
+        uploaded_file = self.request.FILES["csv_file"]
+        rows: list[str] = [
+            line.decode("utf-8").replace("\n", "") for line in uploaded_file
+        ]
+        file_handler = AccessibilityInformationHandler(self.council)
+        file_handler.handle(rows)
+        if file_handler.errors:
+            pass  # do stuff with errors
+        if file_handler.infos:
+            pass  # do stuff with infos
```

Comment:
At the moment there's not reporting of infos (eg duplicate rows) or errors (eg missing ids) to the uploading user. I did find the console output useful in terms of debugging though. My plan had been to that if there were any infos or errors to keep the user on the upload page and display them using `ds-status-error` and `ds-status-message` and the django messages framework.
However I'm leaning towards shoving them on the model, so that the display can be more persistent rather than one time after upload. I will chat this through with @pmk01